ddl/partition.go

```diff
@@ -388,3 +406,15 @@ func checkConstraintIncludePartKey(partkeys []string, constraints map[string]str
 	}
 	return true
 }
+
+// FindRangePartitionColTp find the type of the partitioning key column type.
```

Comment:
s/FindRangePartitionColTp find the type of the partitioning key column type./FindRangePartitionColTp finds the type of the partitioning key column type.